lib/src/directus_api.dart

```diff
@@ -71,9 +71,7 @@ abstract class IDirectusAPI {
   BaseRequest? prepareLogoutRequest();
   bool parseLogoutResponse(Response response);
 
-  Request prepareLoginRequest(String username, String password);
-  Request prepareLoginRequestWithOtp(
-      String username, String password, String otp);
+  Request prepareLoginRequest(String username, String password, String? oneTimePassword);
```

Comment:
Sorry, misread the previous patch request, here is the most important place for the `oneTimePassword` to be made as a named parameter : `{String? oneTimePassword}`. It's public facing, hence should be as clear as possible. That being said, the `prepareLoginRequestMethod` would be improved too by using a named parameter as well.

Reply:
Changed from unnamed optional [String? oneTimePassword] to named optional {String? oneTimePassword}.